pkg/auth/api/api.go

```diff
@@ -344,12 +344,14 @@ func (s *authService) updateUserInfoForOrganizations(
 			OrganizationId: org.Id,
 		})
 		if err != nil {
-			s.logger.Error(
-				"Failed to get account",
-				zap.Error(err),
-				zap.String("email", userInfo.Email),
-				zap.String("organizationId", org.Id),
-			)
+			if status.Code(err) != codes.NotFound {
```

Comment:
```suggestion
                         // Because we don't know what organization the user belongs when exchanging the token,
                         // we ignore logs that were not found to avoid unnecessary logging.
			if status.Code(err) != codes.NotFound {
```